package.json

```diff
@@ -20,7 +20,8 @@
     "test-bbs+": "TEST_SIGNATURE_SCHEME=BBS+ yarn jest",
     "test-ps": "TEST_SIGNATURE_SCHEME=PS yarn jest",
     "test-bbdt16": "TEST_SIGNATURE_SCHEME=BBDT16 yarn jest",
-    "test-all": "TEST_SIGNATURE_SCHEME=BBS yarn jest; TEST_SIGNATURE_SCHEME=BBS+ yarn jest; TEST_SIGNATURE_SCHEME=PS yarn jest; TEST_SIGNATURE_SCHEME=BBDT16 yarn jest"
+    "test-all": "TEST_SIGNATURE_SCHEME=BBS yarn jest; TEST_SIGNATURE_SCHEME=BBS+ yarn jest; TEST_SIGNATURE_SCHEME=PS yarn jest; TEST_SIGNATURE_SCHEME=BBDT16 yarn jest",
```

Comment:
Oh! I actually just realized that this line wasn't changed (only added the comma). Feel free to ignore it then! but I'll leave it here in case it's useful.